src/domain/schema.graphql

```diff
@@ -29,7 +29,6 @@ type DAO @entity {
 	reputationHolders: [ReputationHolder!] @derivedFrom(field: "dao")
 	reputationHoldersCount: BigInt!
 	rewards: [GPReward!] @derivedFrom(field: "dao")
-	register: String!
```

Comment:
why?

Reply:
Because we removed DAORegistry from Arc.

Reply:
DAORegistry was never part of arc . it comes from arc-hive repo https://github.com/daostack/arc-hive/blob/master/contracts/DAORegistry.sol .
we need that.